src/data/content/design-system/fondamenti/bordi-e-raggi.yaml

```diff
@@ -0,0 +1,212 @@
+
+  # KANGAROO: METADATA
+    kangaroo:
+      id: kangarooHero
+      noPadding: true
+      titleSr: Metadati e link per approfondire
+      navposition:
+        items:
+          - title: Ti trovi in
+            tooltip: "La sezione di Designers Italia cui appartiene questa pagina"
+            icon:
+              icon: "sprites.svg#it-folder"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Fondamenti #C
+            url: "../../" #M
+      otherInfo:
+        items:
+          - title: Rispettando
+            tooltip: "Norme e riferimenti che i contenuti di questa pagina ti aiutano a rispettare"
+            icon:
+              icon: "sprites.svg#it-pa"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Linee guida di design #C
+            url: "/norme-e-riferimenti/linee-guida-di-design" #M
+      personalInfo:
+         items: #C
+          - title: Sei
+            tooltip: "Se non ti riconosci in questi profili non preoccuparti: potrai scoprire informazioni utili per saperne di più. In alternativa, naviga la sezione superiore per approfondire il contesto"
+            icon:
+              icon: "sprites.svg#it-user"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Designer, Developer, Qui per curiosità #C
+          - title: Vuoi
+            tooltip: "Obiettivi di progettazione per cui sono utili i contenuti di questa pagina"
+            icon:
+              icon: "sprites.svg#it-flag"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Progettare e sviluppare l'interfaccia utente #C
+      tagsLabel: Parliamo di
+      color: secondary
+      icon:
+        icon: sprites.svg#it-horn
+        size: sm
+        color: secondary
+      tags: [Progettazione interfaccia, Sviluppo interfaccia]
+  
+  sectionsEditorial:
+    - full: true
+      noSpace: false
+      paddingLeft: true
+      title: null
+      text: null
+      background: null
+      components:
+        - name: TextImageCta
+          title: A cosa servono
+          headingLevel: 2
+          text: |
+            ### Bordi
+            I bordi sono linee che delimitano i **contorni di elementi** oppure oggetti presenti in un'interfaccia. 
+            
+            Possono essere applicati a componenti come pulsanti, campi di input, sfondi, immagini o qualsiasi area definita. 
+            
+            I bordi contribuiscono a dare enfasi alla forma e alla struttura di un elemento, possono variare in termini di spessore e stile e creano una separazione fra l'elemento e ciò che lo circonda.
+
+            ### Raggi
+            I raggi definiscono il grado di **curvatura degli angoli** di un elemento che ha un bordo. La misura di un raggio determina quanto gli angoli dell'elemento sono arrotondati.
```

Comment:
Non mi torna la frase "che ha un bordo". I raggi infatti parlano della forma degli angoli di un elemento. Che potrebbe però anche **non avere bordo** inteso come definito sopra. Essere stondato ma senza bordo va bene no? cc @danielaiozzo @zetareticoli

Reply:
sì, capisco cosa vuoi dire, di fatto però la proprietà si chiama "border-radius": non si usa lo stesso parametro anche per smussare angoli senza bordo? cmque lascio a voi

Reply:
Se parli dei bordo in quel senso (potremmo dire come confine dell'elemento) però tutti gli elementi hanno bordi. Non esistono elementi senza, quindi non torna comunque la frase. 
Bisogna chiarire meglio che parliamo di "spessore delle linee" e "raggi degli angoli" per i bordi esterni dell'elemento secondo me.

Reply:
Lo spessore è una proprietà del bordo. 
Il raggio però è una proprietà del confine dell'elemento, potremmo avere un elemento con bordo di spessore 0 e molto arrotondato comunque.

Reply:
Provo a ragionarci e vi faccio una proposta più tardi.

Reply:
ok, cmque credo che il problema sia linguistico, in EN "border" significa "confine" e non necessariamente o esclusivamente "bordo", perciò la frase potrebbe semplicemente essere "I raggi definiscono il grado di **curvatura degli angoli** di un elemento". Ho aggiornato nel frattempo qui e anche più in basso per evitare confusioni

Reply:
Così è corretto. Credo vada però aggiornato anche sotto tutto il blocco Come usarlo a questo punto. Ci vuole un paragrafo dedicato agli spessori del bordo, come usare gli spessori, che manca. E vanno rinominati **bordi** arrotondati e squadrati in **angoli** arrotondati e squadrati probabilmetne no?